kernelci/api/helper.py

```diff
@@ -155,7 +155,7 @@ def create_job_node(self, job_config, input_node,
         try:
             return self._api.node.add(job_node)
         except requests.exceptions.HTTPError as error:
-            raise RuntimeError(json.loads(error.response.text)) from error
+            raise RuntimeError(error.response.text) from error
```

Comment:
API uses `content` to store error messages. See https://github.com/kernelci/kernelci-api/blob/main/api/main.py#L95
```suggestion
            raise RuntimeError(json.loads(error.response.content)) from error
```
See that's how the pipeline parses error message https://github.com/kernelci/kernelci-pipeline/blob/main/src/scheduler.py#L98